src/app/courses/courses.component.ts

```diff
@@ -6,6 +6,7 @@ import { PlanetMessageService } from '../shared/planet-message.service';
 import { MatCheckboxModule } from '@angular/material/checkbox';
 import { SelectionModel } from '@angular/cdk/collections';
 import { Location } from '@angular/common';
```

Comment:
If location is not using anymore, then we can remove import section as well. And also can remove from constructor.